### PR TITLE
Fix_unauthorized_role_access_of_cou_admins

### PR DESCRIPTION
**Pull request by ioigoume**
Fix Calculation of Empty COU access permissions

### CODE REVIEW

File: app/Controller/CoPeopleController.php [hunk 1]
```diff
@@ -801,7 +801,10 @@ public function isAuthorized() {
       $p['cous'] = $roles['admincous'];
     else
       $p['cous'] = array();
-    
+
+    // Pass the roles to the view
+    $p['roles'] = $roles;
```
Comment (lionick) [inline app/Controller/CoPeopleController.php:806]: Is this used somewhere?

**Review by lionick — CHANGES_REQUESTED** [review]

File: app/Controller/CoPeopleController.php (hunk 1 above)
Comment (ioigoume) [reply app/Controller/CoPeopleController.php:806]: @lionick Yes this is used in CoPeople/fields.inc file

**Review by lionick — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**